docs/pages/workflow/debugging.md

```diff
@@ -55,41 +56,65 @@ This might indicate that there is a performance issue. You likely need to run yo
 
 The Expo community and the React and React Native communities are great resources for help when you get stuck. There's a good chance someone else has run into the exact same error as you, so make sure to read the documentation, search the [forums](https://forums.expo.dev/), [GitHub issues](https://github.com/expo/expo/issues/), and [StackOverflow](https://stackoverflow.com/).
 
-## Useful tools for debugging
+## Debugging Tools
 
 Below are a few tools we recommend, and use ourselves, when it comes to debugging your Expo app:
 
-## Developer menu
+### Developer Menu
 
 This menu gives you access to several functions which are useful for debugging, and is built into the Expo Go app. The way you open it is a bit different depending on where you're running the Expo Go app:
 
-- iOS Device: Shake the device a little bit, or touch 3 fingers to the screen.
-- iOS Simulator: Hit <kbd>Ctrl</kbd> + <kbd>Cmd ⌘</kbd> + <kbd>Z</kbd> on a Mac in the emulator to simulate the shake gesture, or press <kbd>Cmd ⌘</kbd> + <kbd>D</kbd>.
-- Android Device: Shake the device vertically a little bit, or run `adb shell input keyevent 82` in your terminal window if your device is connected via USB.
-- Android Emulator: Either hit <kbd>Cmd ⌘</kbd> + <kbd>M</kbd> or <kbd>Ctrl</kbd> + <kbd>M</kbd> or run `adb shell input keyevent 82` in your terminal window.
+- iOS Device: Shake the device, or touch 3 fingers to the screen
+- iOS Simulator: Press <kbd>Ctrl</kbd> + <kbd>Cmd ⌘</kbd> + <kbd>Z</kbd> on a Mac in the emulator to simulate the shake gesture, or press <kbd>Cmd ⌘</kbd> + <kbd>D</kbd>
+- Android Device: Shake the device vertically, or if your device is connected via USB, run `adb shell input keyevent 82` in your terminal
+- Android Emulator: Either press <kbd>Cmd ⌘</kbd> + <kbd>M</kbd> or <kbd>Ctrl</kbd> + <kbd>M</kbd> or run `adb shell input keyevent 82` in your terminal
 
-The Developer Menu gives you a couple different functionalities. A few are pretty self-explanatory, like:
+The Developer Menu looks like this:
 
-- Reload manifest & JS bundle: this will reload your app. Usually this isn't necessary if you have Live or Hot Reload enabled, since it will automatically refresh whenever you save your changes in your text editor.
-- Go to Expo Home: Leave your app and navigate back to the Expo Go app homescreen
-- Enable/Disable Live Reload: When enabled, your app will automatically refresh the JS bundle whenever you save file changes in your project directory.
+<ImageSpotlight alt="The Expo Go Developer Menu, showing the menu options available" src="/static/images/expo-go-developer-menu.png" containerStyle={{ paddingBottom: 0 }} />
+
+The Developer Menu provides multiple options. A few are pretty self-explanatory, like:
+
+- Reload: reload your app - usually not necessary if you have Fast Refresh enabled
```

Comment:
```suggestion
- Reload: reloads your app. Usually not necessary since Fast Refresh is enabled by default
```

Fast refresh is enabled automatically so I think this should give that context.